### ISSUE TITLE
@ts-ignore not supported in <script> tag

### ISSUE THREAD

**Issue by aquinan** [tomblachut/svelte-intellij#172]
When a module is imported in a svelte component's script tag that does not have type definitions, @ts-ignore does not suppress the context error. Also the context menu for the error in WebStorm does not suggest using @ts-ignore.

```svelte
<script lang="ts">
    ...
    // @ts-ignore
    import { Form } from 'svelte-forms-lib';
   ...
</script>
```

**Comment by tomblachut** [comment]
Hi! Thanks for reporting.

@anstarovoyt can this be fixed without implementing LSP?


@aquinan as per your original problem you should write missing declarations instead of escaping from typechecker. It works across your whole project that way..

Below one-liner will suffice and make everything in a module typed as `any`.
```
declare module "svelte-forms-lib";
```

more here https://justintimecoder.com/how-to-polyfil-missing-types-in-typescript/

**Comment by anstarovoyt** [comment]
it must be done on the IDE side. See https://youtrack.jetbrains.com/issue/WEB-48109

**Comment by anstarovoyt** [comment]
WEB-48109 will be fixed in 2020.3
